numpy/core/records.py

```diff
@@ -773,7 +773,48 @@ def fromrecords(recList, dtype=None, shape=None, formats=None, names=None,
 def fromstring(datastring, dtype=None, shape=None, offset=0, formats=None,
                names=None, titles=None, aligned=False, byteorder=None):
     """Create a (read-only) record array from binary data contained in
```

Comment:
```suggestion
    """Create a record array from binary data
    
    Note that despite the name of this function, it does not accept `str` instances.
```

These arrays are not readonly if you pass a writeable bytes-like object.